### PR TITLE
El submit screen

### PR DESCRIPTION
**Pull request by Rian501**
The user can either sign in or register and is immediately 'redirected' (unhidden) to a submit request screen which submits their input along with their profile details (whether just created or stored from login). This eliminates the need for separate pages for sign up or glean submission.

This uses formspree to generate and send email.

This is now basically functional.

To test:
1. Clone repo or fetch branch.
1. Register as a new user and submit a glean request.
1. refresh and sign in again as same user and submit another request.
1. If you have access, check the email address currently set to receive these (or, replace the email in the index file with your own for extensive testing)

### CODE REVIEW

File: glean-request.html [hunk 1]
```diff
@@ -1 +1,55 @@
-<h1>THIS PAGE IS A PAGE</h1>
\ No newline at end of file
+<!doctype html>
```
Comment (mshenfield) [inline glean-request.html]: Is this file still needed?

**Comment by Rian501** [comment]
The firebase file is now included; also I have deleted the files that are no longer needed. If you run this on localhost, it should work. If you run it on [your.ip.address]8080, that might not work as localhost is whitelisted on firebase, and your ip address is not.

Thanks!

**Comment by mshenfield** [comment]
Works for me!
